demisto_sdk/commands/validate/validators/BA_validators/BA127_is_valid_context_path_depth.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Set, Union
+
+from demisto_sdk.commands.common.constants import XSOAR_SUPPORT, GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Command, Integration
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script]
+
+
+class IsValidContextPathDepthValidator(BaseValidator[ContentTypes]):
+    error_code = "BA127"
+    description = "Validate that the level of depth of the context output path in the yml is lower or equal to 5."
+    rationale = "We wish to avoid over nested context to ease on data extraction."
+    error_message = "The level of depth for context output path for {0}: {1} In the yml should be less or equal to 5 check the following outputs:\n{2}"
+    related_field = "contextPath"
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.ADDED]
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+            if content_item.support != XSOAR_SUPPORT:
+                continue
+            if isinstance(content_item, Script):
+                script_paths = self.create_outputs_set(content_item)
+                invalid_paths_str = self.is_context_depth_larger_than_five(script_paths)
+                if invalid_paths_str:
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                "script", content_item.name, invalid_paths_str
+                            ),
+                            content_object=content_item,
+                        )
+                    )
+            else:
+                command_paths = self.create_command_outputs_dict(content_item)
+                invalid_paths_dict = (
+                    self.is_context_depth_larger_than_five_integration_commands(
+                        command_paths
+                    )
+                )
+                if invalid_paths_dict:
+                    final_error = ""
+                    for command, outputs in invalid_paths_dict.items():
+                        final_error += (
+                            self.error_message.format("command", command, outputs)
+                            + "\n"
+                        )
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=final_error,
+                            content_object=content_item,
+                        )
+                    )
```

Comment:
why didn't you change this too? same way we did for the other one

Reply:
wanted to see that we solve the other file.
fixed it